WordPress/Classes/ViewRelated/Blog/SiteManagement/DeleteSiteViewController.swift

```diff
@@ -0,0 +1,279 @@
+                                            comment: "Paragraph 1 of 2 of main text body for the delete screen. NOTE: it is important " +
+                                                     "the localized 'can not' text be surrounded with the HTML '<b>' tags.")
+
+        let paragraph2 = NSLocalizedString("If you're unsure about what will be deleted or need any help, not to worry, " +
+                                           "our support team is here to answer any questions you may have.",
+                                           comment: "Paragraph 2 of 2 of main text body for the delete screen.")
+
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .left
+        let attributes = [ NSFontAttributeName: UIFont.systemFont(ofSize: 17.0),
+                           NSForegroundColorAttributeName: WPStyleGuide.darkGrey(),
+                           NSParagraphStyleAttributeName: paragraphStyle ]
+        let htmlAttributes: StyledHTMLAttributes = [ .BodyAttribute: attributes]
+
+        let attributedText1 = NSAttributedString.attributedStringWithHTML(paragraph1, attributes: htmlAttributes)
+        let attributedText2 = NSAttributedString(string: paragraph2, attributes: attributes)
+
+        let combinedAttributedString = NSMutableAttributedString()
+        combinedAttributedString.append(attributedText1)
+        combinedAttributedString.append(NSAttributedString(string: "\n\r", attributes: attributes))
+        combinedAttributedString.append(attributedText2)
+        sectionThreeBody.attributedText = combinedAttributedString
+
+        let contactButtonAttributes = [ NSFontAttributeName: UIFont.systemFont(ofSize: 17.0),
+                                        NSForegroundColorAttributeName: WPStyleGuide.wordPressBlue(),
+                                        NSUnderlineStyleAttributeName: NSUnderlineStyle.styleSingle.rawValue as AnyObject]
+        supportButton.setAttributedTitle(NSAttributedString(string: NSLocalizedString("Contact Support", comment: "Button label for contacting support"), attributes: contactButtonAttributes),
+                                         for: .normal)
+    }
+
+    /// One time setup of fourth section (delete button)
+    ///
+    fileprivate func setupSectionFour() {
+        let trashIcon = Gridicon.iconOfType(.trash, withSize: CGSize(width: 20.0, height: 20.0)).withRenderingMode(.alwaysTemplate)
+        deleteSiteButton.setTitle(NSLocalizedString("Delete Site", comment: "Button label for deleting the current site"), for: .normal)
+        deleteSiteButton.tintColor = WPStyleGuide.errorRed()
+        deleteSiteButton.setImage(trashIcon, for: .normal)
+    }
+
+    /// Helper function that trims the provided string, prefixes a unicode bullet,
+    /// and sets the proper case.
+    ///
+    fileprivate func styleListItem(_ listItem: String) -> String {
+        return "\u{2022} " + listItem.trim().capitalized
```

Comment:
I guess we could do the localized strings as e.g. `"\u{2022} posts"`, and mention that the unicode character represents a bullet point in a list and should be put in the appropriate position? Otherwise we could detect the current layout direction and either append or prepend the bullet as necessary?